libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/HttpAccess.java

```diff
@@ -187,282 +190,37 @@ private synchronized void setHasNetwork(boolean b) {
         hasNetwork = b;
     }
 
-    /**
```

Comment:
HttpAccess does a lot less than before.
It has methods to return a OkHttpClient or a OkHttpClient.Builder but networking calls are done directly with the OkHttpClient instead of going through HttpAccess methods.